project/index.mjs

```diff
@@ -565,13 +565,95 @@ router.route("/:projectId/layer/:layerId").delete(auth0Middleware(), async (req,
       return respondWithError(res, 400, "Layer not found in project.")
     }
     
-    const response = await layer.deleteLayer(projectId, layerId)
+    await layer.deleteLayer(projectId, layerId)
     res.status(204).send()
   } catch (error) {
     return respondWithError(res, error.status ?? 500, error.message ?? "Error deleting layer from project.")
   }
 })
 
+// Update the Pages in the Layer
```

Comment:
Add some more documentation here around what you can and can't do to the pages in a layer. Specifically, you cannot use this to add new pages into the layer, and it doesn't accept any changes to the page data themselves just the order.